### ISSUE TITLE
Multiple widgets in same model

### ISSUE THREAD

**Issue by ivaniclixx** [madisona/django-google-maps#42]
How to add multiple widgets in the same model?

If the only accepted/hardcoded names are "address" and "geolocation" i see no way to do that.

**Comment by madisona** [comment]
@ivaniclixx You're right - but I'd be happy to accept a pull request if you'd like to help make it more flexible

**Comment by TamaraMystic** [comment]
There's the same problem when it is in Inline
